README.md

```diff
@@ -22,20 +22,27 @@ may change. Even the core `cloud_event_type` annotation may change.
+|Cloud Pub/Sub|[Proto](proto/google/events/cloud/pubsub/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/pubsub/v1/MessagePublishedData.json)|`google.events.cloud.pubsub.v1.MessagePublishedData`|`google.cloud.pubsub.topic.v1.messagePublished`|
+|Cloud Scheduler|[Proto](proto/google/events/cloud/scheduler/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/scheduler/v1/SchedulerJobData.json)|`google.events.cloud.scheduler.v1.SchedulerJobData`|`google.cloud.scheduler.job.v1.executed`|
+|Cloud Storage|[Proto](proto/google/events/cloud/storage/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/storage/v1/StorageObjectData.json)|`google.events.cloud.storage.v1.StorageObjectData`|`google.cloud.storage.object.v1.finalized`<br/>`google.cloud.storage.object.v1.archived`<br/>`google.cloud.storage.object.v1.deleted`<br/>`google.cloud.storage.object.v1.metadataUpdated`|
+|Google Analytics for Firebase|[Proto](proto/google/events/firebase/analytics/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/analytics/v1/AnalyticsLogData.json)|`google.events.firebase.analytics.v1.AnalyticsLogData`|`google.firebase.analytics.log.v1.written`|
+|Firebase Authentication|[Proto](proto/google/events/firebase/auth/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/auth/v1/AuthEventData.json)|`google.events.firebase.auth.v1.AuthEventData`|`google.firebase.auth.user.v1.created`<br/>`google.firebase.auth.user.v1.deleted`|
+|Firebase Realtime Database|[Proto](proto/google/events/firebase/database/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/database/v1/ReferenceEventData.json)|`google.events.firebase.database.v1.ReferenceEventData`|`google.firebase.database.ref.v1.created`<br/>`google.firebase.database.ref.v1.updated`<br/>`google.firebase.database.ref.v1.deleted`<br/>`google.firebase.database.ref.v1.written`|
+|Firebase Remote Config|[Proto](proto/google/events/firebase/remoteconfig/v1/data.proto) / [JSON](https://googleapis.github.io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData.json)|`google.events.firebase.remoteconfig.v1.RemoteConfigEventData`|`google.firebase.remoteconfig.remoteConfig.v1.updated`|
+||||
+<!-- GENERATED END -->
 
-## CloudEvent Type Repos
+A registry of athe JSON schema catalog is located here:
```

Comment:
I think I'd probably reword this part as just:

```text
A [registry of the JSON schema catalog](https://googleapis.github.io/google-cloudevents/jsonschema/catalog.json) is also available.
```

(If you'd prefer not to have that, just remove the "a" at the start of the current "athe".)